setup.py

```diff
@@ -12,6 +12,9 @@
     author="unionai-oss",
     author_email="info@union.ai",
     packages=find_packages(exclude=["tests*"]),
+    package_data={
+        "templates": ["*"],
+    },
```

Comment:
just for my own education, why do we need this?

Reply:
it's to include non-python module files in the package so that e.g. the cookiecutter.json files are included when a user installs unionml